#refecence contains all the needed values for the computation
Pose pose
Velocities dot_pose
Accelerations ddot_pose
